bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -26,6 +26,16 @@
 
 # import libROOTPythonizations with Python version number
 import sys, importlib
+import warnings
+
+if sys.version_info.major < 3:
+    # DeprecationWarning is ignored by default in Python, the next line enables it
+    warnings.filterwarnings("default", category=DeprecationWarning,
+                            module=__name__)
+    warnings.warn(
+        "Support for Python 2 in ROOT is deprecated. It will be removed in "
```

Comment:
Sorry to be picky but wouldn't "The support for Python 2.." with an article read better?